src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -2418,11 +2430,11 @@ class CgroupUtils:
     # 2. If no vnodes are present in the requested resources, try to
     #    span the fewest number of sockets when creating the assignment.
     def assign_job(self, requested, available, node):
-        pbs.logmsg(pbs.EVENT_DEBUG3, "%s: Method called" % (caller_name()))
-        pbs.logmsg(pbs.EVENT_DEBUG3, "Requested: %s" % (requested))
-        pbs.logmsg(pbs.EVENT_DEBUG3, "Available: %s" % (available))
-        pbs.logmsg(pbs.EVENT_DEBUG3, "Numa Nodes: %s" % (node.numa_nodes))
-        pbs.logmsg(pbs.EVENT_DEBUG3, "Devices: %s" % (node.devices))
+        pbs.logmsg(pbs.EVENT_DEBUG4, "%s: Method called" % (caller_name()))
+        pbs.logmsg(pbs.EVENT_DEBUG4, "Requested: %s" % (requested))
+        pbs.logmsg(pbs.EVENT_DEBUG4, "Available: %s" % (available))
+        pbs.logmsg(pbs.EVENT_DEBUG4, "Numa Nodes: %s" % (node.numa_nodes))
+        pbs.logmsg(pbs.EVENT_DEBUG4, "Devices: %s" % (node.devices))
```

Comment:
Why print 4 lines to the log when one would suffice?

Reply:
We still want devices to be on its own line because it could be quite long.